### ISSUE TITLE
Portos e Aeroportos

### ISSUE THREAD

**Issue by JoaoCarabetta** [ipeaGIT/geobr#189]
Seria interessante ter uma base com a loc dos portos e aeroportos. 

Não consegui encontrar uma base com isso, mas desconfio que o ministério da infraestrutura tenha essas infos

**Comment by rdahis** [comment]
Vê se tem aqui: https://www.gov.br/infraestrutura/pt-br/assuntos/dados-de-transportes/bit/bitmulti#nacionais

**Comment by rdahis** [comment]
Aqui parece melhor: https://www.gov.br/infraestrutura/pt-br/assuntos/dados-de-transportes/bit/bitmodosmapas

**Comment by rafapereirabr** [comment]
Obrigado pelas dicas. Eu não acho os dados do www.gov.br bons porque eles não trazem nenhum metadado de órgão produtor, data de atualização, escala espacial etc.  Por isso prefirimos pegar os dados diretamente nas instituições produtora dos dados.


Eu listei algumas fontes no issue #55, que acabei fechando por falta de atividade. Alguns desses dados sao produzidos pelo [DNIT](https://www.gov.br/dnit/pt-br/assuntos/planejamento-e-pesquisa/dnit-geo) mas sei que o IBGE costuma publicar shapes acom algumas dessas informações tambem no ftp deles ftp://geoftp.ibge.gov.br/organizacao_do_territorio/redes_e_fluxos_geograficos/logistica_dos_transportes

**Comment by lucaswarwar** [comment]
Oi pessoal! Dei uma olhada nos Links do MInfra, DNIT e no ftp do IBGE. Pelo que eu vi, os shapes mais atualizados e completos (com os códigos e infos dos portos/aeroportos) são os do Ministério da Infraestrutura. 

Os arquivos do IBGE mais recentes são de 2010, e trazem bem menos detalhe que os do Ministério. Porém, como o @rafapereirabr disse, é preciso confirmar a data de atualização. 

Esbocei um [prep_data ](https://github.com/lucaswarwar/random/blob/master/prep_ports_and_airports.R) para os portos e aeroportos para avaliar os dados, e se for realmente o caso, e os dados forem de boa qualidade, é bastante fácil de expandir para rodovias, hidrovias e ferrovias.

**Comment by rafapereirabr** [comment]
A [ANAC também publica dados dos aeroportos](https://www.anac.gov.br/assuntos/setor-regulado/aerodromos/cadastro-de-aerodromos-civis). Não fica clara a periodicidade de atualização dos dados.

**Comment by lucaswarwar** [comment]
Sim, vou rever a fonte e periodicidade e ver qual é a melhor fonte. Daí atualizo o script do prep tb

**Comment by rafapereirabr** [comment]
Dados de aeroportos podem ser baixados diretamente pelo pacote [flightsbr](https://github.com/ipeaGIT/flightsbr). Isso permite incluir os dados de aeroportos no geobr de maneira muito mais facil. Vou incluir isso no proximo major release do geobr
